doc/sphinx/docs/python/tutorials/nlopt_basics.rst

```diff
@@ -0,0 +1,124 @@
+.. _py_tutorial_nlopt_basics:
+
+A first tutorial on the use of NLOPT solvers
```

Comment:
I capitalise ``NLopt`` throughout the code, following the style on their webpage.